product_docs/docs/pgd/5/reference/conflicts.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Conflict settings
+indexdepth: 2
+deepToC: true
+---
+
+# Conflict detection
+
+## bdr.alter_table_conflict_detection
+
+Allows the table owner to change how conflict detection works for a given table.
+
+### Synopsis
+
+```sql
+bdr.alter_table_conflict_detection(relation regclass,
+                                   method text,
+                                   column_name name DEFAULT NULL)
+```
+
+### Parameters
+
+-   `relation` &mdash; Name of the relation for which to set the new conflict detection method.
+-   `method` &mdash; The conflict detection method to use.
+-   `column_name` &mdash; The column to use for storing the column detection data. This can be skipped, in which case the column name is chosen based on the conflict detection method. The `row_origin` method doesn't require an extra column for metadata storage.
+
+The recognized methods for conflict detection are:
+
+-   `row_origin` &mdash; Origin of the previous change made on the tuple (see [Origin conflict detection](#origin-conflict-detection)). This is the only method supported that doesn't require an extra column in the table.
+-   `row_version` &mdash; Row version column (see [Row version conflict detection](#row-version-conflict-detection)).
+-   `column_commit_timestamp` &mdash; Per-column commit timestamps (described in [CLCD](column-level-conflicts)).
```

Comment:
bad link.... you get the idea....

You need to make these point up and into the actual conflict docs

Reply:
Okay. I am going through the whole doc and double checking links.